snuba/consumers/utils.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import time
+
+from confluent_kafka import KafkaException
+from confluent_kafka.admin import AdminClient
+
+from snuba.utils.streams.configuration_builder import get_default_kafka_configuration
+from snuba.utils.streams.topics import Topic
+
+logger = logging.getLogger("snuba.consumer.utils")
+
+
+class InvalidTopicName(Exception):
+    pass
+
+
+def get_partition_count(topic: Topic) -> int:
+    override_params = {
+        # Default is 60s, do we need that long?
+        "socket.timeout.ms": 2000,
```

Comment:
Why changing this ? If you cannot connect to the broker likely your consumer will not work anyway. I think the timeout that matters is the one you correctly set at line 31.

Reply:
removed it